### PR TITLE
New font size

### PR DESCRIPTION
**Pull request by NickeZ**

### CODE REVIEW

**Comment by conte91** [comment]
Rebase please?

**Comment by NickeZ** [comment]
rebased

File: src/ui/fonts/font_a_9X9.c [hunk 1]
```diff
@@ -23,7 +23,7 @@ static UG_FONT_DATA unsigned char fontBits_font_a_9X9[95][18] = {
   {0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x03,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00 }, // 0x2D '-'
   {0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x02,0x00,0x00,0x00,0x00,0x00 }, // 0x2E '.'
   {0x04,0x00,0x04,0x00,0x02,0x00,0x02,0x00,0x02,0x00,0x01,0x00,0x01,0x00,0x00,0x00,0x00,0x00 }, // 0x2F '/'
-  {0x06,0x00,0x09,0x00,0x09,0x00,0x09,0x00,0x09,0x00,0x09,0x00,0x07,0x00,0x00,0x00,0x00,0x00 }, // 0x30 '0'
+  {0x06,0x00,0x09,0x00,0x09,0x00,0x09,0x00,0x09,0x00,0x09,0x00,0x06,0x00,0x00,0x00,0x00,0x00 }, // 0x30 '0'
```
Comment (conte91) [inline src/ui/fonts/font_a_9X9.c]: Binary font editing made my day.

File: src/ui/components/ui_images.c [hunk 2]
```diff
@@ -20,32 +20,61 @@
...
-                break;
-            case ARROW_UP:
-                UG_DrawPixel(x + w, y + h, C_WHITE);
-                break;
-            default:
-                break;
-            }
+    switch(orientation) {
+    case ARROW_RIGHT:
+        for (int h = 0; h<height; ++h) {
```
Comment (conte91) [inline src/ui/components/ui_images.c]: Spacing/Formatting: `h < height`

File: src/ui/components/confirm_gesture.c [hunk 3]
```diff
@@ -72,7 +72,7 @@ static void _render(component_t* component)
 
     // Draw the top arrow
     const uint16_t padding = 1;
-    x = SCREEN_WIDTH * 15 / 16 - (arrow_height*2 + 1);
+    x = SCREEN_WIDTH * 15 / 16 - (arrow_height + arrow_height/2);
```
Comment (conte91) [inline src/ui/components/confirm_gesture.c]: Spacing

File: src/firmware.c [hunk 4]
```diff
@@ -41,6 +43,27 @@ int main(void)
...
+    //    UG_FontSelect(&font_password_11X10);
+    //    UG_PutString(0, 0, string, 0);
+    //    UG_SendBuffer();
+    //    delay_ms(10000);
+    //    UG_ClearBuffer();
+    //    UG_FontSelect(&font_password_11X12);
+    //    UG_PutString(0, 0, string, 0);
+    //    UG_SendBuffer();
+    //    delay_ms(10000);
+    //}
```
Comment (conte91) [inline src/firmware.c]: Ahem, remove please

**Comment by NickeZ** [comment]
Yeah, for sure the whole thing has to be squashed and clang-formatted. Focus on testing UI for now, so that we can see if the larger font breaks anything or if something looks worse.

File: src/ui/fonts/font_a_9X9.c (hunk 1 above)
Comment (NickeZ) [reply src/ui/fonts/font_a_9X9.c]: :smile:

**Comment by benma** [comment]
Tested, looks great! Will close this and re-open in smaller PRs and with cleanups. Rebased branch with resolved conflicts is here for the time being: https://github.com/benma/bitbox02-firmware/tree/new-font-size

**Pull request closed without merging**